pkg/curatedpackages/packagetypes.go

```diff
@@ -0,0 +1,19 @@
+package curatedpackages
+
+import (
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+)
+
+// DisplayPackage wraps Package to omit undesired members (like Status).
+//
+// This is necessary in part because of https://github.com/golang/go/issues/11939
+// but also because we just don't want to generate a Status section when we're
+// emitting templates for a user to modify.
+type DisplayPackage struct {
+	*api.Package
+	Status *interface{} `json:"status,omitempty"`
```

Comment:
I don't understand adding this if we are trying to omit the `Status` section. Maybe I misunderstand.

Reply:
> I don't understand adding this if we are trying to omit the `Status` section. Maybe I misunderstand.
It's a pointer with omitempty, so it won't be output by the YAML library. Simpler might have been:
```suggestion
	Status interface{} `json:"-"`
```

Reply:
Looks like this doesn't omit the Status object.

Reply:
Based on [the Marshal docs](https://pkg.go.dev/encoding/json#Marshal) it should have, but so long as you have a solution...